server/src/internalClusterTest/java/org/opensearch/cluster/ClusterHealthIT.java

```diff
@@ -439,4 +442,154 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
             completionFuture.actionGet(TimeValue.timeValueSeconds(30));
         }
     }
+
+    public void testHealthAtLevelParamEqualsClusterIsHonoured() {
```

Comment:
Minor - Should we change IT names to reflect the update in the param names in latest changes. So, it comes becomes clear when read in future.

E.g. testHealthWithClusterLevelAppliedAtTransportLayer?

Applicable in all the IT names.

Reply:
Addressed.